addon/helpers/moment-same.js

```diff
@@ -1,5 +1,4 @@
 import Ember from 'ember';
-import moment from 'moment';
 
 export function momentSame([date1, date2, unit]/*, hash*/) {
   if (!date1 || !date2) {
```

Comment:
Yes, we can remove `moment-same-decade` and `moment-same-month` I believe. We also need to PR this helper into ember-moment. 

What does this shim do?